charts/flux2-notification/Chart.yaml

```diff
@@ -2,10 +2,10 @@ apiVersion: v2
 name: flux2-notification
 type: application
 version: 1.8.0
```

Comment:
Can you please bump the version here and the APIs in this chart to `notification.toolkit.fluxcd.io/v1beta2`

Reply:
Bumped - sorry about that